Source/Immutable/Private/ImmutableModule.cpp

```diff
@@ -22,11 +28,61 @@ void FImmutableModule::StartupModule() {
 #endif
 }
 
-void FImmutableModule::ShutdownModule() {
-  // This code will execute after your module is loaded into memory; the exact
-  // timing is specified in the .uplugin file per-module This function may be
-  // called during shutdown to clean up your module.  For modules that support
-  // dynamic reloading, we call this function before unloading the module.
+void FImmutableModule::ShutdownModule()
```

Comment:
readability comment:
Question 1: what's aProcesses, cProcesses, hProcess and cbNeeded? are these common naming convention?
Question 2: would you consider to refactor the heavy nested Ifs to improve the code readability? or would you prefer to roll this out, test it's working by client then we can go back to prefect the code readability?

Reply:
1. These kinda naming may look strange to you but it looks like that due to nature of windows api. They have those kinda of namings. Whoever touched windows api will find it readable.
2. I would prefer to roll out. There is a high potential we will remove this code in future with BLUI updated .